internal/utils/copy/copy_test.go

```diff
@@ -12,31 +12,52 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package copy
+package copy_test
 
 import (
-	"fmt"
 	"os"
 	"path/filepath"
 	"testing"
 
+	"github.com/ZupIT/horusec/internal/utils/copy"
+	"github.com/ZupIT/horusec/internal/utils/testutil"
 	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
 )
 
 func TestCopy(t *testing.T) {
-	t.Run("Should success copy dir", func(t *testing.T) {
-		srcPath, err := filepath.Abs("../../../config")
-		assert.NoError(t, err)
+	src := testutil.GoExample1
+	dst := filepath.Join(t.TempDir(), t.Name())
 
-		dstPath, err := filepath.Abs(".")
-		assert.NoError(t, err)
+	tmpFile, err := os.CreateTemp(os.TempDir(), "test-symlink")
```

Comment:
i think we should decide a standard of temp dirs in test : adopt `t.TempDir()` or `os.TempDir()`. I really don't think there is much difference but i don't like different patterns on the same project

Reply:
Yeah, I totally agree. I prefer t.TempDir() anyway. I'll change this